src/ucp/wireup/address.c

```diff
@@ -384,22 +429,21 @@ static ucs_status_t ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep,
             wiface     = ucp_worker_iface(worker, i);
             iface_attr = &wiface->attr;
 
+            if (!(iface_attr->cap.flags & UCT_IFACE_FLAG_CONNECT_TO_IFACE) &&
```

Comment:
minor: ucs_test_flags may be used to simplify

Reply:
or maybe `if (!(cap_flags & (CONNECT_TO_IFACE|CONNECT_TO_EP)))`